x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/Transform.java

```diff
@@ -213,6 +220,7 @@ protected XPackLicenseState getLicenseState() {
             new ActionHandler<>(GetTransformStatsAction.INSTANCE, TransportGetTransformStatsAction.class),
             new ActionHandler<>(PreviewTransformAction.INSTANCE, TransportPreviewTransformAction.class),
             new ActionHandler<>(UpdateTransformAction.INSTANCE, TransportUpdateTransformAction.class),
+            new ActionHandler<>(SetResetModeAction.INSTANCE, TransportSetResetModeAction.class),
```

Comment:
makes me wonder: all the other actions have _Transform_ in the name, this one not

Reply:
No particular reason. I can rename it